app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -109,7 +109,8 @@ def self.params_for_create
 
   def self.verify_credentials(args)
     kubevirt = raw_connect(args.dig("endpoints", "default")&.slice("server", "port", "token")&.symbolize_keys)
-    kubevirt&.valid? && kubevirt&.virt_supported?
+    kubevirt&.get_api_group_list
+    kubevirt&.get_api_group_kubevirt_io
```

Comment:
TODO I need to confirm this behavior on an cluster that doesn't have kubevirt/osv installed

Reply:
![image](https://github.com/user-attachments/assets/e353ae42-06da-404b-b191-4574fbc72102)

Reply:
Are we still waiting on this verification?

Reply:
No, above screenshot shows it working